app/over_react_redux/todo_client/lib/src/components/app_bar/app_bar_theme_menu.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:over_react/over_react.dart';
+import 'package:over_react/over_react_redux.dart';
+
+import 'package:todo_client/src/actions.dart';
+import 'package:todo_client/src/store.dart';
+import 'package:react_material_ui/react_material_ui.dart';
+import 'package:todo_client/src/components/shared/menu_overlay.dart';
+
+part 'app_bar_theme_menu.over_react.g.dart'; // ignore: uri_has_not_been_generated
+
+class AppTheme {
+  static const String defaultMui = 'defaultMui';
+  static const String typography = 'typography';
+  static const String neonColors = 'neonColors';
+}
+
+UiFactory<AppBarThemeMenuProps> AppBarThemeMenu =
+    connect<AppState, AppBarThemeMenuProps>(
+  mapStateToProps: (state) {
+    return (AppBarThemeMenu()..currentTheme = state.theme);
+  },
+)(_$AppBarThemeMenu); // ignore: undefined_identifier
+
+mixin AppBarThemeMenuPropsMixin on UiProps {
+  String currentTheme;
+}
+
+class AppBarThemeMenuProps = UiProps
+    with MenuOverlayProps, AppBarThemeMenuPropsMixin, ConnectPropsMixin;
+
+class AppBarLocalStorageMenuComponent
+    extends UiComponent2<AppBarThemeMenuProps> {
+  final _overlayRef = createRef<MenuOverlayComponent>();
+
+  @override
+  get consumedProps => propsMeta.allExceptForMixins({MenuOverlayProps});
+
+  @override
+  render() {
+    return (MenuOverlay()
+      ..modifyProps(addUnconsumedProps)
+      ..trigger = (Button()..color = ButtonColor.SECONDARY)('Change Theme')
+      ..useDerivedMaxWidth = true
```

Comment:
I'm seeing this warning in the console when opening the "Change Theme" menu. It looks like it has more to do with the `MenuOverlay` than with changes you made, but just mentioning it here in case you know why the warning is showing up?
<img width="1102" alt="Screen Shot 2020-09-02 at 11 42 31 AM" src="https://user-images.githubusercontent.com/51122966/92023510-b7acbf00-ed11-11ea-9b24-61ad704c1819.png">

Reply:
This error is coming from [the MenuOverlay component](https://github.com/Workiva/over_react/blob/master/app/over_react_redux/todo_client/lib/src/components/shared/menu_overlay.dart#L31) as a result of passing the `useDerivedMaxWidth` prop to the underlying Material UI Menu (which it was recently updated to use). The Material UI component does not support/recognize that prop. 

I've seen a few more of these warnings for other similar props. Since that component/prop logic is used in a couple places, I'm not going to update it for now as part of this PR, but it will be something we'll have to keep in mind if/when we perfect this thing!

Reply:
Sounds good!